### DISCUSSION TITLE
Is the WordPress OSCP plugin working currently?

### DISCUSSION THREAD

**Discussion by ID-EricBorgen** [bullhorn/career-portal#462] [Q&A]
I'm getting the current error in the Career Portal menu under plugins:
____________________________________
When seeking help with this issue, you may be asked for some of the following information:
WordPress version 5.9
Current theme: pro child (version 1.0.0)
Current plugin: Bullhorn-oscp (version 3.3.0)
PHP version 8.0.15
 
 
 
Error Details
=============
An error of type E_ERROR was caused in line 531 of the file /var/www/html/wp-content/plugins/bullhorn-oscp/wp-oscp.php. Error message: Uncaught TypeError: in_array(): Argument #2 ($haystack) must be of type array, string given in /var/www/html/wp-content/plugins/bullhorn-oscp/wp-oscp.php:531
Stack trace:
#0 /var/www/html/wp-content/plugins/bullhorn-oscp/wp-oscp.php(531): in_array()
#1 /var/www/html/wp-admin/includes/template.php(1743): OSCP_Config->oscp_resume_types_callback()
#2 /var/www/html/wp-admin/includes/template.php(1701): do_settings_fields()
#3 /var/www/html/wp-content/plugins/bullhorn-oscp/wp-oscp.php(110): do_settings_sections()
#4 /var/www/html/wp-includes/class-wp-hook.php(307): OSCP_Config->oscp_create_admin_page()
#5 /var/www/html/wp-includes/class-wp-hook.php(331): WP_Hook->apply_filters()
#6 /var/www/html/wp-includes/plugin.php(474): WP_Hook->do_action()
#7 /var/www/html/wp-admin/admin.php(259): do_action()
#8 /var/www/html/wp-admin/plugins.php(10): require_once('...')
#9 {main}
thrown

**Comment by charlesabarnes** [comment]
Hello, 

The plugin should be working. Looks like the error on that line relates to the accepted resume types.  Please ensure you have a type selected. Thanks.  If you have any further issues, please contact bullhorn support to ensure this is properly debugged. 

Thank you!

**Comment by ID-EricBorgen** [comment]
It is erroring at that precise point:
![image](https://user-images.githubusercontent.com/69921086/153945675-f1a7db00-9fd6-4c8c-b09f-0911cb98066a.png)

I can only speculate as I'm not a developer by trade, but maybe something that PHP was once more flexible about is now enforced in PHP8 that I'm running?

**Reply by charlesabarnes** [reply]
oooooh thats a good guess, I have not tested PHP 8 yet.  If you submit a ticket through support it will get me time to refactor/test the plugin for that. Thanks for that info though!

**Reply by charlesabarnes** [reply]
@ID-EricBorgen I've been doing some investigation on this issue.  I began to reproduce the same issue after forcing it into this state.   I was able to fix it after deleting option `oscp_config_data` from the `wp_options` table.     I suggest doing the same for the interim.  I am still investigating how it was possible to get into this state.
